<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>queryStu</class>
 <widget class="QDialog" name="queryStu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>查询学生</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="2,8">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,10,1,0">
       <item>
        <widget class="QComboBox" name="cbbMethod">
         <item>
          <property name="text">
           <string>选择查询方式</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>按姓名</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>按学号</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>按院系</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leCnt"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnSearch">
         <property name="text">
          <string>开始查询</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnClear">
         <property name="text">
          <string>清除信息</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
